ecolex/legislation.py

```diff
@@ -198,12 +203,19 @@ def harvest_file(upfile):
             except Exception:
                 logger.debug('Error parsing legYear %s' % (legYear))
 
-        url_value = document.attrs.get('url', None)
-        if url_value:
-            legislation['legLinkToFullText'] = url_value
+        filenames = get_content(document.findAll('link_to_full_text'))
+        url_values = []
+        for filename in filenames:
+            if filename.endswith('.doc'):
+                path = "http://extwprlegs1.fao.org/docs/texts/"
+            elif filename.endswith('.html'):
+                path = "http://extwprlegs1.fao.org/docs/html/"
+            else:
+                path = "http://extwprlegs1.fao.org/docs/pdf/"
```

Comment:
The suggestion above applies here as well. I would add these URLs to `settings.py` and I would get them from env variables, so that they can be changed without changing the code.